lib/ecstatic.js

```diff
@@ -124,13 +124,26 @@ var ecstatic = module.exports = function (dir, options) {
               url: parsed.pathname + '.' + defaultExt + ((parsed.search)? parsed.search:'')
             }, res, next);
           }
-          else {
-            // Try to serve default ./404.html
+          else if (!handleError) {
+            // If we're not handling errors/fallbacks, bail out now
+            middleware({
+              url: req.url,
+              statusCode: 404
+            }, res, next);
+          }
+          else if (path.basename(parsed.pathname, defaultExt) === '200.') {
```

Comment:
This check seems brittle and awkward, and I don't think anyone else does it. Do you see a way around it? I don't, least not without a bigger refactor of ecstatic around passing state (which I want to do, but is kind of a whole thing).

I'll have to think about this.